.gitignore

```diff
@@ -1,2 +1,3 @@
 node_modules/
 temp/
+.idea
```

Comment:
Actually, you were right. The `_gitignore` was modified but `.gitignore` was not.